pcl_utilities/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>pcl_utilities</name>
+  <version>0.1.0</version>
+  <description>Implements Various Pointcloud Filtering Services</description>
```

Comment:
Maybe change filtering to processing - since there are only a couple of filtering nodes